src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Statistics/BoundaryLevelViewModel.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.Statistics;
+
+public class BoundaryLevelViewModel
+{
+    public long Id { get; set; }
+
+    public string Level { get; set; }
```

Comment:
Rather than using `string` here I would change the type to be `GeographicLevel`. That way it's more obvious what the values of `Level` are going to be and it constrains it to those values rather than an arbitrary string.

I think you will need to annotate it like so

```csharp
[JsonConverter(typeof(StringEnumConverter), typeof(CamelCaseNamingStrategy))]
public required GeographicLevel Level { get; init; }
```